### ISSUE TITLE
calabash should automatically accept new VPN configurations

### ISSUE THREAD

**Issue by jmoody** [calabash/calabash-ios#1279]
Reported by @joyzoursky

Having the same problem here on VPN Configurations springboard alert:

ios version: 10.2.1
run-loop version: 2.3.0
calabash version: 0.20.4

springboard info: {{"title"=>"“MyApp” Would Like to Add VPN Configurations", "buttons"=>["Allow", "Don’t Allow"], "tries"=>20, "error"=>"A SpringBoard alert is blocking test execution and it cannot be dismissed."}

The only commands that works when the alert pops are `springboard_alert` and `springboard_alert_visible?`. Other commands like `touch`, `query`, or `touch_point` are also not working.
```
calabash-ios 0.20.4> device_agent.springboard_alert
[
    [0] {
          "enabled" => true,
               "id" => "“MyApp” Would Like to Add VPN Configurations",
          "hitable" => true,
             "rect" => {
                 "y" => 244.5,
                 "x" => 52.5,
            "height" => 178.5,
             "width" => 270
        },
        "hit_point" => {
            "x" => 187.5,
            "y" => 333.75
        },
            "label" => "“MyApp” Would Like to Add VPN Configurations",
             "type" => "Alert"
    }
]
```

In my case, it works for some time before (although not very stable), but one day it keeps occurring and cannot dismiss the alert anymore. Also, this is the second time that I face this problem. In the first time, I upgraded it from iOS 9.3.2 to iOS 10.2.1 to resolve this (still happening after reset phone). But after running tests for a period of time it happens constantly again.

**Comment by jmoody** [comment]
https://jira.xamarin.com/browse/TCFW-1073

**Comment by joyzoursky** [comment]
@jmoody Would this fix force user to allow the configuration every time? As we might also like to demonstrate user tapping the disallow button in some tests.

**Comment by jmoody** [comment]
Yes.

But we are working on allowing users to disable automatic SpringBoard alert handling.

Xcode  8.3 has turned out to be HUGE time sink so everything is delayed.

**Comment by jmoody** [comment]
Fixed in run-loop 2.4.0
